### ISSUE TITLE
How do you compare two dictionaries??

### ISSUE THREAD

**Issue by jthough7** [tegge-classroom/STAT2984-2018#11]
I am just so stuck!!! I really want to know what these 2 dictionaries have in common:(

food = {"ham" : "yes", "egg" : "yes", "spam" : "no" }
my_dict = {1: 'apple', 2: 'ball'}

**Comment by apappy56** [comment]
I do not think that these two dictionaries have anything in common Jonathan! Maybe the apple and the other food have something in common but i do not thing ball means anything to ham egg and spam!
